programs/psinode/main.cpp

```diff
@@ -1920,12 +2091,36 @@ void run(const std::string&              db_path,
                          result = std::make_shared<OpenSSLProver>(key.service);
                       }
                    }
-                   std::vector<Claim> existing;
-                   prover->get(existing);
+                   else
```

Comment:
Looks like this else statement assumes there's a key device, but `key.device.empty()` could still be empty